tests/testsuite.src/run_extensions.at

```diff
@@ -5070,6 +5070,54 @@ AT_CHECK([$COBCRUN_DIRECT ./prog], [0],
 AT_CLEANUP
 
 
+AT_SETUP([COBOLX format])
+AT_KEYWORDS([extensions runmisc])
+
+AT_DATA([prog.cob], [
+       >>SOURCE FORMAT COBOLX
+ IDENTIFICATION DIVISION.
+ PROGRAM-ID. prog.
+* blah blah blah
+ PROCEDURE DIVISION.
+                                                        DISPLAY "Hello!"
+     .
+ END PROGRAM prog.
```

Comment:
Is "END PROGRAM" GCOS style? If not please remove that part and test it (as below) using `-fformat`. Please also test the debugging line for cobolx.